README.md

```diff
@@ -5,34 +5,34 @@
 * [Prerequisities](#prerequisites)
 
 
-# Demo list
+# What is this repo?
 
-There are multiple demos in this repo that showcase Kafka stream processing on the Confluent Platform.
-Some demos run on local Confluent Platform installs (download [Confluent Platform](https://www.confluent.io/download/)) and others run on Docker (install [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/)).
+This repo contains a collection of demos that showcase Kafka stream processing on the Confluent Platform.
+Some of the demos run on local Confluent Platform installs (download [Confluent Platform](https://www.confluent.io/download/)) and others run on Docker (install [Docker](https://docs.docker.com/install/) and [Docker Compose](https://docs.docker.com/compose/install/)). All of them contain READMEs that tell you what each demo does and how to run it, and all of them are tested against the latest Confluent Platform release.
 
-## Where to start
+## What should I do with it?
 
-The best demo to start with is [cp-demo](https://github.com/confluentinc/cp-demo), which spins up a Kafka streaming ETL using KSQL for stream processing.
-cp-demo also comes with a playbook and video series, and is a great configuration reference for Confluent Platform.
+1. You should browse the demos, see which one sounds interesting, then run it!
```

Comment:
I generally prefer "Browse the demos..," instead of "You should browse the demos..."   Did you intentionally write the latter?